hypha/apply/determinations/models.py

```diff
@@ -99,7 +99,7 @@ class Determination(DeterminationFormFieldsMixin, AccessFormData, models.Model):
         on_delete=models.PROTECT,
     )
 
-    outcome = models.IntegerField(verbose_name=_("Determination"), choices=DETERMINATION_CHOICES, default=1)
+    outcome = models.IntegerField(verbose_name=_("Determination"), choices=DETERMINATION_CHOICES)
```

Comment:
Let's not do any db level changes?

Looks like the issue only requires UI/forms level changes.

_changing default would require us to have a discussion on why it was set in the first place..and what could be the implications of changing it now_